lib/iex/lib/iex/helpers.ex

```diff
@@ -543,7 +555,10 @@ defmodule IEx.Helpers do
     case File.ls(path) do
       {:ok, items} ->
         sorted_items = Enum.sort(items)
-        ls_print(path, sorted_items)
+        print_function = fn(item, opts, width) -> 
+          format_item(Path.join(opts[:path], item), String.pad_trailing(item, width)) 
```

Comment:
We don't need to pass the opts argument. You can already access the `path` here as part of the closure.